armi/utils/__init__.py

```diff
@@ -837,6 +837,49 @@ def safeCopy(src: str, dst: str) -> None:
     runLog.extra("Copied {} -> {}".format(src, dst))
 
 
+def safeMove(src: str, dst: str) -> None:
```

Comment:
Can we get a new unit test for this new function?

If it helps I see the test go something like this:

* Use the `TemporaryDirectoryChanger` to create a temp dir for your test
* create two dirs, one called `src` and one called `dst`
* add a blank fie or two inside `src` (or just add 1 character of text inside them)
* test 1: use this function to move the files from `src/` into `dst`
* test 2: use this function to move the entire `src/` dir into `dst`

Something like that, anyway.

Reply:
I added some unit tests